CONTRIBUTING.md

```diff
@@ -50,20 +71,18 @@ Check out the [Go-Filecoin code overview](CODEWALK.md) for a brief tour of the c
 - Major changes should have a [Design Doc](https://github.com/filecoin-project/designdocs).
 - For minor changes, file an issue for your change if it doesn't yet have one, and outline your implementation plan on the issue.
 
-### Pull Requests
+### Pull Requests and Reviews
 
-- Try to keep PRs small, no more than 400 lines or 8 files.
-- Always squash commits.
-- For now, you'll need to request write access (via chat) to create a PR because CircleCI doesn't play well with private branches. Once we open source this repo, forking is preferred.
-- Committers should merge their own PRs after Approval, because they have the most context.
-
-### Code Reviews
+We review every change before merging to master, using GitHub pull requests.
+**Try to keep PRs small**, no more than 400 lines or 8 files. Code reviews are easier, faster, and more effective when the diff is small.
 
 `go-filecoin` requires 2 approvals for all PRs. We use the following process, which aims to merge code quickly and efficiently while avoiding both accidental and malicious introduction of bugs, unintended consequences, or vulnerabilities.
 
-The first review is done by a committer familiar with that area of the codebase. Once they deem it ready for merge, they will assign a maintainer for the second review. Once the PR has 2 Approvals and blocking comments have been addressed, the committer rebases and merges the PR.
+1. The first review is done by a committer familiar with that area of the codebase. Once they deem it ready for merge, they will assign a maintainer for the second review. 
+1. Once the PR has 2 Approvals and blocking comments have been addressed, the committer [rebases and merges](#landing-changes) the PR.
+    - If the author is also a committer, they merge their own PRs.
```

Comment:
`master` branch is currently protected, I think the only minimal number of people have merge access to it.

I think the group of commiters is much wider than a group of people with master merge permission.

Reply:
> `master` branch is currently protected, I think the only minimal number of people have merge access to it.

@filecoin-project/go-filecoin-collaborators have write, but not merge access. This group is fairly broad, includes many people with a history of useful PRs.
@filecoin-project/go-filecoin-committers have merge access to master. This group is quite limited.

@anorth's wording is technically correct, but @Kubuxu is it confusing to you when you read it? Should we say "If the author has merge access as a committer or maintainer, they merge their own PRs"?

Reply:
Thanks @mishmosh your wording is more robust, I'll use that.